addons/binding/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/hw/HwDiscoveryService.java

```diff
@@ -23,18 +33,20 @@ public HwDiscoveryService(HwSerialBridgeHandler handler) {
 
     @Override
     protected void startScan() {
-        // Scan for dimmers
-        try {
-            for (int m = 1; m <= 8; m++) { // Modules
-                for (int o = 1; o <= 4; o++) { // Outputs
-                    String address = String.format("[01:01:00:%02d:%02d]", m, o);
-                    handler.sendCommand("RDL, " + address);
-                    Thread.sleep(5);
+        new Thread(() -> {
+            try {
+                logger.debug("Starting scan for HW Dimmers");
+                for (int m = 1; m <= 8; m++) { // Modules
+                    for (int o = 1; o <= 4; o++) { // Outputs
+                        String address = String.format("[01:01:00:%02d:%02d]", m, o);
+                        handler.sendCommand("RDL, " + address);
+                        Thread.sleep(5);
+                    }
                 }
+            } catch (InterruptedException e) {
+                logger.warn("Scan interrupted");
             }
-        } catch (InterruptedException e) {
-            logger.warn("Scan interrupted");
-        }
+        }).start();
```

Comment:
Generally you should be using the scheduler from AbstractDiscoveryService to schedule background tasks.  Sorry if my earlier comment led you astray.